components/post_view/post_attachment_opengraph/post_attachment_opengraph.jsx

```diff
@@ -282,6 +282,29 @@ export default class PostAttachmentOpenGraph extends React.PureComponent {
             this.loadImage(imageUrl);
         }
 
+        let body;
+        if (data.description || imageUrl) {
+            let separator;
+            if (data.description && imageUrl) {
+                separator = ' &nbsp';
+            }
+
+            body = (
+                <div>
```

Comment:
Why do we have this `div` wrapper?

Reply:
I can see how it looks without it, but I just used what we had before